### DISCUSSION TITLE
Change in the computation of the scatter index?

### DISCUSSION THREAD

**Discussion by elmedhi** [DHI/modelskill#164] [General]
Hi, 
I noticed that @jsmariegaard changed the formula of the scatter index in the skill table.  (https://github.com/DHI/fmskill/commit/1d55986c74ab38251c89f8a124102747946561aa). 

I have very different results now in some scatter plots, with the same data. For example, 
This plot was made in September/October: 
![image](https://user-images.githubusercontent.com/125252219/219592879-93fc29d5-b4db-436d-bd79-6a2ad788c50e.png)
This one was made a few days ago: 
![image](https://user-images.githubusercontent.com/125252219/219592936-272a85c5-62c8-4891-a6ae-8a896a2a0357.png)

Could you explain the reason of this modification?

Thank you!

**Comment by jsmariegaard** [comment]
Hi @elmedhi , it was per request from @daniel-caichac-DHI following this issue: https://github.com/DHI/fmskill/issues/137 - the work was carried out in the PR https://github.com/DHI/fmskill/pull/142 - does this provide enough information?

**Reply by elmedhi** [reply]
Yes, thank you :)

**Reply by daniel-caichac-DHI** [reply]
@elmedhi  the old implementation is not the default anymore but you can get it by using scatter_index2
[def scatter_index2(obs: np.ndarray, model: np.ndarray) -> float:](https://github.com/DHI/fmskill/blob/a575d9615f455559d9cb686ac30f1a1af0c48106/fmskill/metrics.py#L430)
